lookout/style/format/feature_utils.py

```diff
@@ -61,6 +61,18 @@ def __eq__(self, other: "VirtualNode") -> bool:
                 and self.y == other.y
                 and self.path == other.path)
 
+    def is_labeled_on_lines(self, lines: Optional[Set[int]]) -> bool:
+        """
+        Return true for labeled VirtualNode instance that located on specified lines.
+
+        :param lines: list of lines or None. None is considered as all possible lines.
+        :return: Condition value
+        """
+        if self.y is None:
+            return False
+        return (len(set(range(self.start.line, self.end.line + 1)) & lines) != 0
```

Comment:
return lines is None or lines.intersection(range(self.start.line, self.end.line + 1))

Reply:
ok,
```
return lines is None or bool(set(range(self.start.line, self.end.line + 1)) & lines)
```
just to always have bool value.

Reply:
Not needed.

Reply:
And please use lines.intersection since it doesn't create a useless set

Reply:
>And please use lines.intersection since it doesn't create a useless set

oh, sorry. I missed that. 

> Not needed.

but I have a typo hint `-> bool`
do you want my to change it to `-> Union[bool, Set]`?
I'd prefer to keep `bool(...)`

Reply:
```python
In[8]: False or set()
Out[8]: set()
```

Reply:
as you want, this will be rebased into one inlined call where the bool won't be present anyway.